fx-src/FirefoxStorage.js

```diff
@@ -108,12 +108,16 @@ const currentSchemaVersion = 1;
 export default class FirefoxAdapter extends BaseAdapter {
   constructor(collection, options={}) {
     super();
+    const {sqliteHandle=null} = options;
     this.collection = collection;
-    this._connection = null;
+    this._connection = sqliteHandle;
     this._options = options;
   }
 
-  _init(connection) {
+  // We need to be capable of calling this from "outside" the adapter
+  // so that someone can initialize a connection and pass it to us in
+  // adapterOptions.
+  static _init(connection) {
```

Comment:
If this is intended to be called from outside, it's part of the public API and shouldn't be denoted as private with an underscore in the name. Also, have you considered moving this as a separate function? Maybe the FFx module export system isn't flexible enough to make usage painless?